x-pack/plugins/cross_cluster_replication/public/app/store/actions/auto_follow_pattern.js

```diff
@@ -135,9 +120,10 @@ export const deleteAutoFollowPattern = (id) => (
       }
 
       // If we've just deleted a pattern we were looking at, we need to close the panel.
-      const detailPanelAutoFollowPatternName = getDetailPanelAutoFollowPatternName(getState());
-      if (detailPanelAutoFollowPatternName && response.itemsDeleted.includes(detailPanelAutoFollowPatternName)) {
-        dispatch(closeAutoFollowPatternDetailPanel());
+      const ids = arrify(id);
+      const autoFollowPatternId = getSelectedAutoFollowPatternId(getState());
+      if (ids.includes(autoFollowPatternId)) {
```

Comment:
In the logic above this point, we're surfacing both error toasts (for when an auto-follow pattern fails to be deleted) and success toasts (for when an auto-follow pattern is deleted). So isn't this `onSuccess` callback invoked when there is a partial failure during a bulk delete? If this is this case, the original logic results in a better UX in the event that the auto-follow pattern in the detail panel has failed to be deleted, because we're checking `response.itemsDeleted` instead of `ids`.